app/_src/gateway/kong-enterprise/secrets-management/secrets-rotation.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Secrets Rotation
+---
+
+Secrets rotation is a process of updating secrets. Regular rotations of secrets is considered a good
+practice. Here are some reasons why secrets rotation is important:
+
+* Reducing impact of compromised secrets
+* Enhancing resilience against brute-force attacks
+* Complying with security regulations
+* Maintaining separation of duties
+* Adapting to evolving threats
+* Mitigating the effects of insider threats
+
+Kong needs to hold many types of secrets to be able to process requests, and to integrate with other
+systems and clients:
+
+* TLS certificates
+* Credentials and API keys (to access databases, identity providers, and other services)
+* Cryptographic keys (digital signing and encryption)
```

Comment:
Like in the code, I would propose that we use "rotate" to refer to the process of changing secret values on a schedule or in response to a disclosure even, and "renew" to refer to the mechanism inside of Kong that causes it to re-read secret values from their respective vaults.  What is described above is "rotation", while what is described below is "renewal".

It should also be mentioned that Kong caches secrets, as that is why the renewal mechanism is required in the first place.  It might even make sense to call the mechanism "refresh" as it is similar to OIDC refresh, so there is a precedent for using that word.

It may also be worth mentioning that in the context of vaults, the term `TTL` refers to the time that a secret is cached inside of Kong.  The term is not defined and then suddenly used, and it makes sense to provide a narrow definition to make the documentation easier to understand.